client/src/Games/DrawTheWord/DrawingBoard.jsx

```diff
@@ -0,0 +1,264 @@
+/**
+ * @Resources
+ * https://dev.to/jerrymcdonald/creating-a-shareable-whiteboard-with-canvas-socket-io-and-react-2en
+ * https://stackoverflow.com/questions/2368784/draw-on-html5-canvas-using-a-mouse
+ */
+import React, { useState, useEffect, useRef } from "react";
+import io from "socket.io-client";
+
+import "./DrawingBoardStyles.css";
+
+/**
+ * 
+ * @param {any} props 
+ * @returns This function will return Timer, Guessing Word, White Board and colour pallet for 'Draw the Word' Game.
+ */
+export default function DrawingBoard(props) {
+
+  const SERVER = "http://localhost:3001";
+  let socket;
+  const [timeoutValue, setTimeoutValue] = useState(undefined);
+  const colorsRef = useRef(null);
+  const colourPalletDict = {
+    black: "#000000",
+    white: "#ffffff",
+    darkGray: "#6e6e6e",
+    lightGray: "#cccccc",
+    maroon: "#4f0000",
+    brown: "#854500",
+    red: "#db0000",
+    lightpink: "#fcbdf6",
+    orange: "#fc8200",
+    lightOrange: "#ffc60d",
+    yellow: "#fff700",
+    lightYellow: "#fffca1",
+    green: "#198000",
+    lightGreen: "#32ff00",
+    blue: "#0011ff",
+    lightBlue: "#00ecf0",
+    indigo: "#271069",
+    lightIndigo: "#8c93d1",
+    purple: "#6e00e3",
+    lightPurple: "#d9c5f0",
+  };
+
+  useEffect(() => {
+    socket = io(SERVER);
+
+    socket.on("connection", () => {});
```

Comment:
Might need to do some refactoring here. The specific "room" that the connection is in is based on the "roomCode". I think this establishes a new connection / room.